soc/arm/nuvoton_numaker/m46x/soc.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/init.h>
+#include <zephyr/kernel.h>
```

Comment:
not needed

Reply:
Not need <zephyr/init.h>.
Keep <zephyr/kernel.h> for DT_NODE_HAS_PROP(DT_NODELABEL(xxx), ...)

Reply:
this is DT API, not Kernel.

Reply:
#include <zephyr/device.h> also could resolve the above error.
For DT API, should I include <zephyr/device.h> instead of <zephyr/kernel.h> ?